files/hermit

```diff
@@ -22,7 +22,17 @@ export HERMIT_EXE=${HERMIT_EXE:-${HERMIT_STATE_DIR}/pkg/hermit@${HERMIT_CHANNEL}
 
 if [ ! -x "${HERMIT_EXE}" ]; then
   echo "Bootstrapping ${HERMIT_EXE} from ${HERMIT_DIST_URL}" 1>&2
-  curl -fsSL "${HERMIT_DIST_URL}/install.sh" | /bin/bash 1>&2
+  INSTALL_SCRIPT="$(mktemp)"
+  # This value must match that of the install script
+  INSTALL_SCRIPT_SHA256="HERMIT_INSTALL_SCRIPT_SHA256"
+  # Install script is versioned by the first 10 characters of its sha256sum
+  INSTALL_SCRIPT_VERSION=$(echo "${INSTALL_SCRIPT_SHA256}" | head -c 10)
```

Comment:
Using ten (10) characters from the hash should be sufficient to prevent name collisions. According to the [birthday paradox](https://en.wikipedia.org/wiki/Birthday_problem), the probability of a truncated hash collision in this case is around `2^{-20} = 9.53674316e-7`. Given that the install script will probably rarely change, we should be good in practice.

Reply:
Why truncate though? Are we pressed for storage?

Reply:
It's mostly to avoid the potentially visually unpleasant long file names. Note that a full sha256 hash is represented by 64 hex characters. I'm open to use the full hash value if the file name length concern is unfounded.

Reply:
I like it, cleaner.